src/test/java/io/jenkins/pluginhealth/scoring/probes/LastCommitDateProbeTest.java

```diff
@@ -0,0 +1,83 @@
+package io.jenkins.pluginhealth.scoring.probes;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Map;
+
+import io.jenkins.pluginhealth.scoring.model.Plugin;
+import io.jenkins.pluginhealth.scoring.model.ProbeResult;
+import io.jenkins.pluginhealth.scoring.model.ResultStatus;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class LastCommitDateProbeTest {
+    @Test
+    public void shouldKeepScmAsKey() {
+        assertThat(new LastCommitDateProbe().key()).isEqualTo("last-commit-date");
+    }
+
+    @Test
+    public void shouldNotRequireRelease() {
+        assertThat(new LastCommitDateProbe().requiresRelease()).isFalse();
+    }
+
+    @Test
+    public void shouldHaveValidDescription() {
+        assertThat(new LastCommitDateProbe().getDescription()).isNotEqualTo(new Probe() {
+            @Override
+            protected ProbeResult doApply(Plugin plugin) {
+                return null;
+            }
+
+            @Override
+            public String key() {
+                return null;
+            }
+        }.getDescription());
+    }
+
+    @Test
+    public void shouldBeExecutedAfterSCMLinkValidation() {
+        assertThat(SCMLinkValidationProbe.ORDER).isLessThan(LastCommitDateProbe.ORDER);
+    }
+
+    @Test
+    public void shouldReturnSuccessStatusOnValidSCM() {
+        final Plugin plugin = mock(Plugin.class);
+        final LastCommitDateProbe probe = new LastCommitDateProbe();
+
+        when(plugin.getDetails()).thenReturn(Map.of(SCMLinkValidationProbe.KEY, ProbeResult.success("scm", "The plugin SCM link is valid")));
+        when(plugin.getScm()).thenReturn("https://github.com/jenkinsci/parameterized-trigger-plugin.git");
+        final ProbeResult r = probe.apply(plugin);
+
+        assertThat(r.id()).isEqualTo("last-commit-date");
+        assertThat(r.status()).isEqualTo(ResultStatus.SUCCESS);
+    }
```

Comment:
Because this would validate the deletion of the file and all, I think it would be better to have an integration test.

Reply:
Sure, I'm on it.